packages/ui/Label/stories.tsx

```diff
@@ -0,0 +1,40 @@
+// src/Label.stories.tsx
+import type { Meta, StoryObj } from "@storybook/react";
+import { Label } from "./index";
+
+const meta: Meta<typeof Label> = {
+  title: "UI/Label",
+  component: Label,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "centered",
+    docs: {
+      description: {
+        component: `
+The **Label** component is used to label form fields in your application. 
```

Comment:
The **Label** component is used to label the form fields.

Can you please update the description? Also, please add a description in other existing stories as well. 😊